tests/helpers/graphical_units.py

```diff
@@ -47,6 +47,11 @@ def set_test_scene(scene_object, module_name):
         tests_directory, "control_data", "graphical_units_data"
     )
     print(path_control_data)
+    image_without_alpha = data[:, :, :3]
+    if (
+        sum(sum(sum(image_without_alpha)))
+    ) == 0:  # adds all color channels in 2 dimensions, and checks if the image is empty.
+        raise Warning("The image is empty, something went wrong")
```

Comment:
I don't understand what you did here. Could you clarify what are you trying to do, and more precisely what does the indexing thing mean? (like, what is the shape of the data, etc, I don't remember lol)  


And, why not using a built-in feature of numpy to check if all elements are all zeros, like `np.all()`?

Reply:
oh yes, I just came up with the most simple solution I could think of, but  np.all() is definitely more elegant!
The indexing is removing the alpha channel, so what remains is only the RBG image.
And this warning tells you that something in set_test_scene might went wrong because you only get an black image.